plugins/dashboard_pi/src/baro_history.cpp

```diff
@@ -284,7 +284,10 @@ void DashboardInstrument_BaroHistory::DrawForeground(wxGCDC* dc)
   m_ratioW = double(m_DrawAreaRect.width) / (BARO_RECORD_COUNT-1);
  // dc->DrawText(wxString::Format(_(" Max %.1f Min %.1f since %02d:%02d  Overall Max %.1f Min %.1f "),m_MaxPress,m_MinPress,hour,min,m_TotalMaxPress,m_TotalMinPress), m_LeftLegend+3+2+degw, m_TopLineHeight-degh+5);
  // Cant get the min sice to work...
- dc->DrawText(wxString::Format(_(" Max %.1f since %02d:%02d  Overall Max %.1f Min %.1f "),m_MaxPress,hour,min,m_TotalMaxPress,m_TotalMinPress), m_LeftLegend+3+2+degw, m_TopLineHeight-degh+5);
+  wxString sMax = _( "Max" ), sSince = _( "since" ), sOverall = _( "Overall" ), sMin = _( "Min" );
```

Comment:
Ok Pavel.
It was a suggest to avoid user errors, assuming they are not aware of c-code in the strings. My solution was taken from similar examples in the core code. 
The wordings Min, Max etc are not mine. I haven't changed anything like that. Just adapted what's already there.
So what's your suggest to solve it according to your  "we should get rid of it in the codebase," ?
Håkan